<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonalInfosDialog</class>
 <widget class="QDialog" name="PersonalInfosDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>个人资料</string>
  </property>
  <widget class="QPushButton" name="ok_pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_pushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="switchHeadPushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>80</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>头像</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>261</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>账号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>昵称</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>性别</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>生日</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>邮箱</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>简介</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>上次登陆时间</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="accountLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nickNameLineEdit"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="sexManRadioButton">
          <property name="text">
           <string>男</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="sexWomanRadioButton">
          <property name="text">
           <string>女</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDateEdit" name="birthdayDateEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mailLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="remarkLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lastLoginTimeLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
